core/src/main/scala/kafka/security/auth/ResourceType.scala

```diff
@@ -18,27 +18,41 @@ package kafka.security.auth
 
 import kafka.common.{BaseEnum, KafkaException}
 import org.apache.kafka.common.protocol.Errors
+import org.apache.kafka.common.resource.{ResourceType => JResourceType}
 
-sealed trait ResourceType extends BaseEnum { def error: Errors }
-
-case object Cluster extends ResourceType {
-  val name = "Cluster"
-  val error = Errors.CLUSTER_AUTHORIZATION_FAILED
+sealed trait ResourceType extends BaseEnum {
+  def error: Errors
+  def toJava: JResourceType
 }
 
 case object Topic extends ResourceType {
   val name = "Topic"
   val error = Errors.TOPIC_AUTHORIZATION_FAILED
+  val toJava = JResourceType.TOPIC
 }
 
 case object Group extends ResourceType {
   val name = "Group"
   val error = Errors.GROUP_AUTHORIZATION_FAILED
+  val toJava = JResourceType.GROUP
+}
+
+case object Cluster extends ResourceType {
+  val name = "Cluster"
+  val error = Errors.CLUSTER_AUTHORIZATION_FAILED
+  val toJava = JResourceType.CLUSTER
+}
+
+case object Broker extends ResourceType {
```

Comment:
I don't have a strong preference.  I just kept it here since it is in the Scala code, and I wanted to be consistent.  If we are removing it, though, we should do it now.